### PR TITLE
Bug/5950

### PR DESCRIPTION
**Pull request by utkarshgautamnyu**
Edited describe-settings.json and DomainServerSettingsManager.cpp so that Adding a Path does not require a server restart. 

Testing Notes:
Build and Run the PR
Launch the interface and domain-server
go to localhost:40100 -> Settings -> Add a new Path -> Save.
Check the domain-server logs, it should show NO restart. 
( Sample Line to Check:  [07/05 13:43:05] [DEBUG] [default] domain-server is restarting. **This line should NOT be there in the logs**)
Check the interface when adding a path. The interface should NOT go *Not Connected* for sometime when the new path is added.

### CODE REVIEW

File: domain-server/resources/describe-settings.json [hunk 1]
```diff
@@ -50,11 +50,12 @@
     {
       "label": "Places / Paths",
       "html_id": "places_paths",
+	  "restart": false,
```
Comment (birarda) [inline domain-server/resources/describe-settings.json]: this indentation is off, should be in line with the line above

File: domain-server/resources/describe-settings.json [hunk 2]
```diff
@@ -50,11 +50,12 @@
...
       "settings": [
         {
           "name": "paths",
           "label": "Paths",
-          "help": "Clients can enter a path to reach an exact viewpoint in your domain.<br/>Add rows to the table below to map a path to a viewpoint.<br/>The index path ( / ) is where clients will enter if they do not enter an explicit path.",
+		  "help": "Clients can enter a path to reach an exact viewpoint in your domain.<br/>Add rows to the table below to map a path to a viewpoint.<br/>The index path ( / ) is where clients will enter if they do not enter an explicit path.",
```
Comment (birarda) [inline domain-server/resources/describe-settings.json]: indentation is off here, should be over 1 space farther

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1355/HighFidelity-Beta-PR10887-f824290a4cdc5ec34ce9a372afae7ff477989fa5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1355/HighFidelity-Beta-PR10887-f824290a4cdc5ec34ce9a372afae7ff477989fa5.exe

**Comment by CainFoool** [comment]
Testing shortly.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1401/HighFidelity-Beta-PR10887-d5c88bc6de495ff14147ef08b8717a44af583fa2.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1401/HighFidelity-Beta-PR10887-d5c88bc6de495ff14147ef08b8717a44af583fa2.dmg

**Pull request merged**